card/mod/01_core/set/all/rules.rb

```diff
@@ -1,45 +1,73 @@
-RuleSQL = %{
+# frozen_string_literal: true
+
+RULE_SQL = %(
   select rules.id as rule_id, settings.id as setting_id, sets.id as set_id,
     sets.left_id as anchor_id, sets.right_id as set_tag_id
   from cards rules
   join cards sets     on rules.left_id  = sets.id
   join cards settings on rules.right_id = settings.id
-  where sets.type_id     = #{Card::SetID }    and sets.trash     is false
+  where sets.type_id     = #{Card::SetID}     and sets.trash     is false
   and   settings.type_id = #{Card::SettingID} and settings.trash is false
   and                                             rules.trash    is false
   and   (settings.codename != 'follow' or rules.db_content != '');
-}
+).freeze
+
+# FIXME - "follow" hardcoded above
```

Comment:
Annotation keywords like `FIXME` should be all upper case, followed by a colon, and a space, then a note describing the problem.